modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/disruptor/inbound/StateEventHandler.java

```diff
@@ -107,6 +108,11 @@ public void updateSlotsAndQueueCounts(InboundEventContainer eventContainer) {
             // wise.
             messagingEngine.incrementQueueCount(message.getMetadata().getDestination(), 1);
 
+            //Tracing Message
+            MessageTracer.trace(message, MessageTracer.SLOT_INFO_UPDATED);
+
+            eventContainer.pubAckHandler.ack(message.getMetadata());
```

Comment:
This pub ack handler is not relevant here. removed in PR https://github.com/wso2/andes/pull/192